src/main/resources/twilio-python/context.handlebars

```diff
@@ -1,6 +1,6 @@
 class {{apiName}}Context(InstanceContext):
 
-    def __init__(self, version: Version{{#instancePathParams}}, {{paramName}}: {{{dataType}}}{{/instancePathParams}}):
+    def __init__(self, version: Version{{#instancePathParams}}, {{paramName}}: {{#if allowableValues.enumVars}}{{instanceName}}.{{/if}}{{{dataType}}}{{/instancePathParams}}):
```

Comment:
I think this other PR fixes this: https://github.com/twilio/twilio-oai-generator/pull/463

Reply:
Noted. Will revert the changes